### PR TITLE
Update the sample navidrome.service for use in Arch Linux

### PR DESCRIPTION
**Pull request by sumnerevans**
I'm not sure if I should create a new one, or just update this one.

### CODE REVIEW

File: contrib/navidrome.service [hunk 1]
```diff
@@ -28,8 +31,3 @@ RestrictRealtime=yes
 SystemCallFilter=~@clock @debug @module @mount @obsolete @privileged @reboot @setuid @swap
 ReadWritePaths=/opt/navidrome/
 PrivateDevices=yes
-ProtectSystem=full
-ProtectHome=true
```
Comment (jvoisin) [inline contrib/navidrome.service:32]: Why remove those two lines?

File: contrib/navidrome.service [hunk 2]
```diff
@@ -28,8 +31,3 @@ RestrictRealtime=yes
 SystemCallFilter=~@clock @debug @module @mount @obsolete @privileged @reboot @setuid @swap
 ReadWritePaths=/opt/navidrome/
```
Comment (jvoisin) [inline contrib/navidrome.service:32]: Shouldn't this be changed as well?

**Comment by deluan** [comment]
I think the files in the `contrib` should be seen as samples. Maybe we should transfer them to the documentation site? What do you think?

File: contrib/navidrome.service (hunk 1 above)
Comment (sumnerevans) [reply contrib/navidrome.service:32]: Apparently I was too tired...

File: contrib/navidrome.service (hunk 2 above)
Comment (sumnerevans) [reply contrib/navidrome.service]: Yep. I should stop trying to code in the middle of the night...

**Comment by sumnerevans** [comment]
So, I kinda want to keep it in source control and in sync with the releases because of the way that AUR packages work. Basically, I want the source for the `.service` file to be something like:

```
    https://raw.githubusercontent.com/deluan/navidrome/v0.15.0/contrib/navidrome.service
```

That way, I can use an MD5 checksum on that file because it's pegged to the release tag. I don't want to just pull off of some documentation site that may get out of sync with the release. I also don't want to have to manually go and change it in the AUR package every time somebody changes it in the docs.

**Comment by deluan** [comment]
@sumnerevans So maybe the `navidrome.service` file should be in the AUR repo itself, no?

**Comment by sumnerevans** [comment]
That's the thing, I don't want to have to go and manually update it whenever the sample `navidrome.service` changes... I guess I can add "retrieve the sample from the docs" to my update script, but that just seems like a PITA and vulnerable to issues like somebody changing the sample to be no longer compatible with the existing one.

**Pull request merged**